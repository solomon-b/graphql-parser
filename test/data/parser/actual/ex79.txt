[ DefinitionTypeSystem
    ( TypeSystemDefinitionType
        ( IOTDef
            ( InputObjectTypeDefinition
                { _iotDescription = Nothing
                , _iotName = "Point2D"
                , _iotDirectives = Nothing
                , _iotValueDefinitions =
                    [ InputValueDefinition
                        { _ivdDescription = Nothing
                        , _ivdName = "x"
                        , _ivdType = NamedType "Float"
                        , _ivdDefaultValue = Nothing
                        , _ivdDirectives = Nothing
                        }
                    , InputValueDefinition
                        { _ivdDescription = Nothing
                        , _ivdName = "y"
                        , _ivdType = NamedType "Float"
                        , _ivdDefaultValue = Nothing
                        , _ivdDirectives = Nothing
                        }
                    ]
                }
            )
        )
    )
]