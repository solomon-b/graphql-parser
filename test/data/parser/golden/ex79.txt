[ DefinitionTypeSystem
    ( TypeSystemDefinitionType
        ( IOTDef
            ( InputObjectTypeDefinition
                { _iotSpan = Span
                    { _start = AlexSourcePos
                        { _line = 0
                        , _col = 1
                        }
                    , _end = AlexSourcePos
                        { _line = 3
                        , _col = 2
                        }
                    }
                , _iotDescription = Nothing
                , _iotName = "Point2D"
                , _iotDirectives = Nothing
                , _iotValueDefinitions = Just
                    ( InputFieldsDefinition
                        { _ifdSpan = Span
                            { _start = AlexSourcePos
                                { _line = 0
                                , _col = 15
                                }
                            , _end = AlexSourcePos
                                { _line = 3
                                , _col = 2
                                }
                            }
                        , _ifdDefinition = InputValueDefinition
                            { _ivdSpan = Span
                                { _start = AlexSourcePos
                                    { _line = 1
                                    , _col = 3
                                    }
                                , _end = AlexSourcePos
                                    { _line = 1
                                    , _col = 11
                                    }
                                }
                            , _ivdDescription = Nothing
                            , _ivdName = "x"
                            , _ivdType = NamedType
                                ( Span
                                    { _start = AlexSourcePos
                                        { _line = 1
                                        , _col = 6
                                        }
                                    , _end = AlexSourcePos
                                        { _line = 1
                                        , _col = 11
                                        }
                                    }
                                ) "Float"
                            , _ivdDefaultValue = Nothing
                            , _ivdDirectives = Nothing
                            } :|
                            [ InputValueDefinition
                                { _ivdSpan = Span
                                    { _start = AlexSourcePos
                                        { _line = 2
                                        , _col = 3
                                        }
                                    , _end = AlexSourcePos
                                        { _line = 2
                                        , _col = 11
                                        }
                                    }
                                , _ivdDescription = Nothing
                                , _ivdName = "y"
                                , _ivdType = NamedType
                                    ( Span
                                        { _start = AlexSourcePos
                                            { _line = 2
                                            , _col = 6
                                            }
                                        , _end = AlexSourcePos
                                            { _line = 2
                                            , _col = 11
                                            }
                                        }
                                    ) "Float"
                                , _ivdDefaultValue = Nothing
                                , _ivdDirectives = Nothing
                                }
                            ]
                        }
                    )
                }
            )
        )
    )
]